kie-server-parent/kie-server-tests/pom.xml

```diff
@@ -26,9 +26,10 @@
     <kie.server.context>kie-server-services</kie.server.context>
     <cargo.container.id>valid-id-needs-to-be-specified-in-appropriate-container-profile</cargo.container.id>
     <kie.server.controller.context>kie-server-controller-services</kie.server.controller.context>
-    <kie.server.classifier>ee7</kie.server.classifier>
+    <kie.server.classifier>ee8</kie.server.classifier>
+    <kie.server.controller.classifier>ee7</kie.server.controller.classifier>
```

Comment:
I think this is clear now, but maybe we could have here a note/comment about why we have these 2 slightly similar properties for classifier?